# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/yaml-cpp/hotfix-stdint.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

src/emitterutils.cpp:221:11: error: 'uint16_t' was not decl
ared in this scope
  221 | std::pair<uint16_t, uint16_t> EncodeUTF16SurrogatePair(int codePoint) {
      |           ^~~~~~~~

--- yaml-cpp-0.8.0/src/emitterutils.cpp.vanilla	2025-06-27 22:24:37.184043281 +0200
+++ yaml-cpp-0.8.0/src/emitterutils.cpp	2025-06-27 22:24:45.438063257 +0200
@@ -1,6 +1,7 @@
 #include <algorithm>
 #include <iomanip>
 #include <sstream>
+#include <inttypes.h>
 
 #include "emitterutils.h"
 #include "exp.h"
